src/enums/dissolutionTypesNames.ts

```diff
@@ -0,0 +1,44 @@
+// *** TODO: update these according to table in ticket
```

Comment:
This depends on what values the API uses. The sample data I have is from COLIN, I think, and I suspect LEAR will only use a subset of these.

In the meantime, merging this code probably (*) won't hurt anything.

(*) The Create UI will need the same enums. Actually I was thinking about moving these to BCRS Shared Components except these aren't finalized yet.